modules/oe_editorial_entity_version/src/Form/NodeRevisionRevertForm.php

```diff
@@ -157,6 +157,12 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $this->revision->setChangedTime($this->time->getRequestTime());
     $this->revision->set('moderation_state', 'draft');
 
+    // With took over translation, the changed time should be
```

Comment:
Carry over translations is not a functionality of this module so we shouldn't mention it. The comment could be updated to 
```suggestion
    // Set the same changed time for existing translations of the current revision.
```

Reply:
fixed